plugin/src/main/kotlin/tanvd/grazi/GraziLibResolver.kt

```diff
@@ -0,0 +1,11 @@
+package tanvd.grazi
+
+import tanvd.grazi.ide.ui.components.dsl.msg
+import tanvd.grazi.language.Lang
+
+object GraziLibResolver {
```

Comment:
I would better move it in some package `remote` or `jar`?

Reply:
Done